apps/challenges/views.py

```diff
@@ -16,10 +25,18 @@
 from participants.models import Participant, ParticipantTeam
 from participants.utils import get_participant_teams_for_user, has_user_participated_in_challenge
 
-
-from .models import Challenge, ChallengePhase, ChallengePhaseSplit
+from .models import Challenge, ChallengePhase, ChallengePhaseSplit, ChallengeConfiguration
 from .permissions import IsChallengeCreator
-from .serializers import ChallengeSerializer, ChallengePhaseSerializer, ChallengePhaseSplitSerializer
+from .serializers import (ChallengeConfigurationSerializer,
+                          ChallengePhaseSerializer,
+                          ChallengePhaseSplitSerializer,
+                          ChallengeSerializer,
+                          DatasetSplitSerializer,
+                          LeaderboardSerializer,
+                          ZipConfigurationChallengeSerializer,
+                          ZipFileCreateChallengePhaseSplitSerializer,)
```

Comment:
`ZipFileCreateChallengePhaseSplitSerializer` is too long. Can we please shortened it and still keep it as meaningful?